_includes/guides.html

```diff
@@ -22,7 +22,7 @@ <h5 class="grid_content grid_content_dark">Overview</h5>
                         <a href="https://rackhd.readthedocs.io/en/latest/tutorials/index.html" class="list-group-item list-group-item-fixed-border">
                              <h5 class="grid_content grid_content_dark">Tutorials</h5>
                         </a>
-                        <a href="https://rackhd.readthedocs.io/en/latest/rackhd/ubuntu_source_installation.html" class="list-group-item list-group-item-fixed-border">
+                        <a href="https://rackhd.atlassian.net/wiki/display/RAC1/RackHD+Installation+Guide+-1.0.0" class="list-group-item list-group-item-fixed-border">
```

Comment:
will do in the future. because the 1.0.0 page seems been published in twitter by Tom.C?
so currently, renaming it is not wise at this moment ...